diff --git a/mkspecs/common/macx.conf b/mkspecs/common/macx.conf
index 61a1300860..0db79330ac 100644
--- a/mkspecs/common/macx.conf
+++ b/mkspecs/common/macx.conf
@@ -6,7 +6,7 @@ QMAKE_PLATFORM         += macos osx macx
 QMAKE_MAC_SDK           = macosx

 QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.13
-QMAKE_APPLE_DEVICE_ARCHS = x86_64
+QMAKE_APPLE_DEVICE_ARCHS = arm64

 # Should be 10.15, but as long as the CI builds with
 # older SDKs we have to keep this.
